js/main.js

```diff
@@ -0,0 +1,48 @@
+// query selecting the input element type text that holds the value of the anime title entered by the user
+var $animeTitle = document.querySelector('#anime');
+// a function to create a new XMLHttpRequest to get anime data based on the string value of the input element above
+var animeDataReq = new XMLHttpRequest();
+function getReq(title) {
+  animeDataReq.open('GET', 'https://api.jikan.moe/v4/anime?letter=' + title);
+  animeDataReq.responseType = 'json';
+  animeDataReq.send();
+}
```

Comment:
I suggest structuring your code around making a new `XMLHttpRequest` each time you need to make a request. Otherwise, you'll run into problems later if you attempt to re-use one.